bugs-fixed/split-fs-from-array.awk

```diff
@@ -1,5 +1,5 @@
 BEGIN {
-        a[1] = "elephantie"
+        a[1] = "elephantine"
```

Comment:
This is the only change which I consider particularly interesting. To make it, I had to shift the output check as well.